### PR TITLE
Add GPU GEMV

### PR DESCRIPTION
**Pull request by vlad-nazarov**

### CODE REVIEW

File: include/oneapi/internal/math/reference_gemv.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+/** @ingroup oneapi_internal
+ * @{
+ */
+
+/**
+ *  <a name="DAAL-CLASS-ONEAPI-INTERNAL__REFERENCEGEMV"></a>
+ *  \brief Adapter for reference GEMV routine
+ */
+template <typename algorithmFPType>
+struct DAAL_EXPORT ReferenceGemv
```
Comment (PetrovKP) [inline include/oneapi/internal/math/reference_gemv.h:51]: You need to implement reference by based GEMM (use GEMM for GEMV for reference case).

**Comment by vlad-nazarov** [comment]
/intelci: run
set infra_branch=dev/vnazarov-gemv

**Comment by vlad-nazarov** [comment]
/intelci: run
set infra_branch=dev/vnazarov-gemv

**Pull request closed without merging**